config/routes.rb

```diff
@@ -0,0 +1,19 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  root 'homepages#index'
+
+  resources :homepages, only: [:index]
```

Comment:
Given how your site is designed, I might not even include the route for `/homepages`, and use the root route everywhere.